content/doc/developer/publishing/wiki-page.adoc

```diff
@@ -49,3 +49,36 @@ If a non-wiki URL is specified, for example to a GitHub repository, it will only
 Given the limitations inherent in including scraped content, many advanced macros will not show up properly on the plugin site:
 Basically every macro that uses AJAX to load data in the background will not work.
 Since Jenkins will link to the plugin's page on the plugin site from its plugin manager, this will result in a bad experience reading broken documentation for many users.
+
+== Migrating from Wiki to GitHub
```

Comment:
This seems to be copied from `CONTRIBUTING.adoc`. If so, the original content should be deleted as part of this PR, and made to link here instead.

Reply:
Nope. There is some copy-paste, but there will be more difference between guidelines while I move forward with tools